docs/devops.md

```diff
@@ -47,7 +46,7 @@ DevOps principles are applied to and assisted by the following 3 Key areas all w
 - **Processes** – DevOps builds further upon Lean Agile, applying concepts of lean manufacturing to IT value streams.
 - **Tools** – DevOps relies on enhanced tools & technologies, including automation, telemetry, & collaboration tools.
 
-## What Is the Goal of DevOps?
+## What is the goal of DevOps?
```

Comment:
title case?

Reply:
@xavier506 then remove the question mark, right?